src/modules/universal-discovery/components/create/create.choose.content.type.component.js

```diff
@@ -7,19 +7,50 @@ export default class ChooseContentTypeComponent extends Component {
     constructor(props) {
         super(props);
 
+        let selected = {};
+        const isForcedContentType = props.allowedContentTypes.length === 1;
+        const allowedIncludesPreselected = props.allowedContentTypes.includes(props.preselectedContentType);
```

Comment:
`hasPreselectedAllowedContentType`

Reply:
Is it possible, that in props you get a preselected content type which is not allowed?

Reply:
It comes from backend configurations in yml. I don't trust backend ;)

Reply:
If you cannot believe data provided by your own backend, then it's a bad sign. If the backend was some kind of third-party, then it would be ok.

Reply:
In the yml configuration might be anything, it's defined by end-developer.

Reply:
I'm not sure we should do it this way. If integrator does a mistake, then it's an integrator's fault not ours. What do you think @Nattfarinn and @lserwatka ?